src/dhtproto/client/UsageExamples.d

```diff
@@ -162,8 +162,12 @@ version ( UnitTest )
                     this.log.error(this.msg_buf);
                     break;
 
-                default:
```

Comment:
I wonder about:
```D
template DefaultCase ( )
{
  istring DefaultCase = "version(D_Version2){} else { default: assert(false); }";
}
```

Reply:
Yeah, this looks fine enough. Maybe the naming could be more descriptive.